auto_scan.py

```diff
@@ -113,6 +114,48 @@ def get_container_registry_domains(lw_client):
     return registry_domains
 
 
+def get_inline_scanner_access_token(lw_client):
+    logger.info('Looking for existing Inline Scanner...')
+
+    response = lw_client.container_registries.search(json={
+        'filters': [
+            {
+                'expression': 'eq',
+                'field': 'name',
+                'value': SCANNER_INTEGRATION_NAME
+            }
+        ]
+    })
+
+    for integration in response.get('data', []):
+        logger.info('Inline Scanner found.')
+        logger.debug(json.dumps(integration))
+        return integration.get('serverToken', {}).get('serverToken')
+
+    logger.info('Inline Scanner not found.')
+
+
+def create_inline_scanner_integration(lw_client):
+    logger.info('Creating new Inline Scanner...')
+
+    response = lw_client.container_registries.create(
+        name=SCANNER_INTEGRATION_NAME,
+        type='ContVulnCfg',
```

Comment:
This is more a note for the SDK -- instead of `ContVulnCfg` as a random string, maybe we can expose Constants or Types within the SDK? So it'd instead be something like `ContainerRegistryTypes.ContainerVulnConfig` or something? I think the string is fine in this context, but for the average user of the SDK we should probably type the inputs for discoverability.